app/serializers/api/v1/message_serializer.rb

```diff
@@ -5,10 +5,18 @@ class MessageSerializer < ApplicationSerializer
     embed :ids, include: true
 
     attributes :id, :body, :state, :is_private, :created_at,
-      :updated_at, :offer_id, :item_id
+      :updated_at, :offer_id, :item_id, :order_id, :designation_id
```

Comment:
I'll have to test it. Will get back to you on this. @steveyken

Reply:
@steveyken, I checked if we can remove designation_id from message_serializer or not. 
It stops working, because on stock we have designation, and on browse we have order.
if we send order_id instead of designation_id, Ember won't recognize it and wont create associations between designation and messsage, resulting in faliure of autoreload. 

cc - @namrataukirde